Automattic/wp-calypso
# Jetpack Manage: add Get Help nav to overview page

Resolves Automattic/jetpack-manage#150

## Proposed Changes

This adds the "Get Help" links to the overview page.

Technical scope: pf36In-mf-p2#get-help

![image](https://github.com/Automattic/wp-calypso/assets/32492176/6453774c-3499-42ea-8d36-b03158dd356a)

## Testing Instructions

* Review the code.
* Verify the `expanded` state is preserved upon refresh.
* Verify tracks events fire when clicking a link.
* Verify tracks events fire when expanding/collapsing the component.

## Pre-merge Checklist
